src/OscapScannerRemoteSsh.cpp

```diff
@@ -107,10 +107,28 @@ void OscapScannerRemoteSsh::evaluate()
         return;
     }
 
-    emit infoMessage(QObject::tr("Querying capabilities on remote machine..."));
-
     {
         SshSyncProcess proc(mSshConnection, this);
+        emit infoMessage(QObject::tr("Checking if oscap is available on remote machine..."));
+
+        proc.setCommand(QString("hash"));
```

Comment:
This relies on /bin/bash being the interpreter for the user. We may want to go with something more portable. Some people use zsh or even just sh.